flink-contrib/flink-storm-examples/src/main/java/org/apache/flink/storm/util/FileSpout.java

```diff
@@ -38,6 +38,8 @@
 	protected String path = null;
 	protected BufferedReader reader;
 
+	protected boolean finished;
+
 	public FileSpout() {}
```

Comment:
`FileSpout` should be implemented the Storm way -- to set the "finished"
flag here does not make sense from a Storm point of view (there is no
such thing as a finite spout)